service/turn.rest/src/test/java/org/kaazing/gateway/service/turn/rest/TurnRestIT.java

```diff
@@ -67,18 +67,14 @@
                                 .requireRole("username")
                             .done()
 
-                            .property("generate.credentials", "class:" + DefaultCredentialsGenerator.class.getName())
                             .property("key.alias", "turn.shared.secret")
                             .property("key.algorithm", "HmacSHA1")
-                            .nestedProperty("uris")
-                                .property("uri", "uri1")
-                                .property("uri", "uri2")
-                                .property("uri", "uri3")
-                            .done()
-                            .nestedProperty("options")
-                                .property("credentials.ttl", "43200")
-                                .property("username.separator", ":")
-                            .done()
+                            .property("credentials.generator", "class:" + DefaultCredentialsGenerator.class.getName())
+                            .property("credentials.ttl", "43200")
+                            .property("username.separator", ":")
+                            .property("url", "url1")
```

Comment:
please change the url1, url2, url3 to be real examples:

for example
"turn:1.2.3.4:9991?transport=udp",
       "turn:1.2.3.4:9992?transport=tcp",
       "turns:1.2.3.4:443?transport=tcp"

as from Spec https://tools.ietf.org/html/draft-uberti-rtcweb-turn-rest-00